lib/build_pack/downloader.rb

```diff
@@ -0,0 +1,32 @@
+module BuildPack
+  class Downloader
+    MYSQL_BASE_URL = "http://security.debian.org/pool/updates/main/m/mysql-5.5/"
+    REGEX = /.*(mysql-client-5\.5_5\.5\...-0\+deb.u._amd64.deb).*/
```

Comment:
I'm glad you asked!

1. The build pack wasn't failing gracefully if we didn't find a suitable client, so I've added support for that.
2. The client naming convention is confusing, so I've put an example match above the regex:
 - `mysql-client-5.5_5.5.52-0+deb8u1_amd64.deb`
https://github.com/Shopify/heroku-buildpack-mysql/pull/5/files#diff-7b49d9a22af7543841ac03f29aa0b075R7 
 - the regex does currently match any minor version, but I've made it more restrictive to expect digits as the minor version (it was accepting any characters as the minor version).